src/quilla/browser/browser_validations.py

```diff
@@ -116,6 +119,12 @@ def validate(self) -> List[BaseReport]:
         except Exception as e:
             # Catch exception and crash the program, but clean up
             # the browser first
+            self.ctx.logger.error(
```

Comment:
If we do provide suppress_exceptions option, why are we not checking it here?

Reply:
The suppress exception tries to catch every known error from performing the steps, so the only time that this block of code should ever execute should be when the user passes `--debug` to Quilla. When we're not suppressing exceptions though, we still need to make sure that the browser is terminated before finishing the execution, so this handles that.